app/javascript/app/service-provider-form.js

```diff
@@ -76,4 +76,8 @@ $(function(){
       preview.textContent = logo_file.name;
     });
   }
+
+  $(document).ready(function() {
```

Comment:
Looks like the JS here is always checking this. Is there a reason to do that with JS instead of when we render the form?